TeamLBAC: Add scenario 6 to the docs (#87215)
grafana/grafana: docs/sources/administration/data-source-management/teamlbac/create-teamlbac-rules/index.md

A user that is part of Team A and part of Team B will have access to logs that match `namespace="dev"`.

A user that is not part of Team A and part of Team B, that is `Editor` or `Viewer` will have access to all logs (due to the query permission for the user).

### Task 6: User A is Admin and part of Team B

> We have team B, user A is part of Team B and has an `Admin`.

Reviewer: Nit: maybe specify that you're talking about org admin and not a team admin here?
```suggestion
We have team B, user A is part of Team B and has an `Admin` basic role.
```

- Team B has no roles assigned

> - Team B has Query permissions to Datasource Loki
> 
> - Team B has rule `{ project_id="project-something-102461" }`
> 
> User A may only query Datasource Loki where `{ project_id="project-something-102461" }` and no other logs on the data source.

Reviewer: nit: some grammar/style suggestions
```suggestion
- Team B has Query permissions to data source Loki

- Team B has a rule `{ project_id="project-dev" }`

User A may only access logs for data source Loki that match `{ project_id="project-dev" }` and no other logs on the data source.
```
